### PR TITLE
Use better release-artifact names.

### PR DESCRIPTION
**Pull request by floitsch**
## Summary by CodeRabbit

- **New Features**
	- Enhanced installation logic for Toit, supporting additional operating systems and architectures.
	- Improved error messaging for unsupported architectures and version checks.

- **Bug Fixes**
	- Clarified version handling logic to ensure accurate installation paths and URLs.

- **Chores**
	- Updated CI workflow to include macOS 13 and specified version updates.

### CODE REVIEW

**Comment by floitsch** [comment]
@coderabbitai review

File: action.yml [hunk 1]
```diff
@@ -32,16 +32,57 @@ runs:
       run: |
         TOIT_VERSION=${{ inputs.toit-version }}
 
-        if [[ "$RUNNER_OS" = "Linux" ]]; then
-          TOIT_FILE=toit-linux.tar.gz
-        elif [[ "$RUNNER_OS" = "macOS" ]]; then
-          TOIT_FILE=toit-macos.tar.gz
-        elif [[ "$RUNNER_OS" = "Windows" ]]; then
-          TOIT_FILE=toit-windows.tar.gz
+        ARTIFACT_NAMING_THRESHOLD_VERSION=v2.0.0-alpha.163
```
Comment (kasperl) [inline action.yml]: Want this to be 164 too?

Comment (kasperl) [inline action.yml]: ASSETS?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: action.yml (hunk 1 above)
Comment (floitsch) [reply action.yml]: done.

Comment (floitsch) [reply action.yml]: done.

**Pull request merged**